airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncKubePodStatus.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.process;
+
+public enum AsyncKubePodStatus {
+  NOT_STARTED, // Pod hasn't been started yet.
```

Comment:
Is it used, I don't see the status being written at all.

Reply:
It's used for testing only atm.